### DISCUSSION TITLE
Read only access

### DISCUSSION THREAD

**Discussion by engegu** [six7/figma-tokens#317] [Q&A] [answered]
We are in the planning stages of a new design system project.

Our Source of truth will be style dictionary. From there we will generate a JSON file, maybe multiple, that we would like designers to have synced, read-only access to. What would be the easiest way to achieve this currently?

Also, what's the easiest way to link FT to an existing JSON file in a Github Repo or locally stored

Thanks in advance for any help @six7

**Comment by six7** [comment, marked as answer]
> We are in the planning stages of a new design system project.
> 
> Our Source of truth will be style dictionary. From there we will generate a JSON file, maybe multiple, that we would like designers to have synced, read-only access to. What would be the easiest way to achieve this currently?
> 
> Also, what's the easiest way to link FT to an existing JSON file in a Github Repo or locally stored
> 
> Thanks in advance for any help @six7

To have a read-only sync you can use the URL sync feature: https://docs.tokens.studio/sync/url - you could add gitlab or github there and then provide credentials in the header field. Or use the GitHub sync and set your designers to `Read` access on the repository, the plugin is able to see that and only provide Read permissions

the json file should look something like this one: https://github.com/six7/figma-tokens-example/blob/ab707d557b04b7eac72e368a09eff9b397472bb0/tokens.json

hope this helps!
